src/aioamazondevices/api.py

```diff
@@ -1134,3 +1140,19 @@ async def set_do_not_disturb(self, device: AmazonDevice, state: bool) -> None:
         await self._session_request(
             method="PUT", url=url, input_data=payload, json_data=True
         )
+
+    async def set_volume(
+        self,
+        device: AmazonDevice,
+        volume: int,
+    ) -> None:
+        """Call Alexa.DeviceControls.Volume to set volume."""
+        if volume < MIN_VOLUME:
+            _vol_clean = MIN_VOLUME
+        if volume > MAX_VOLUME:
+            _vol_clean = MAX_VOLUME
+        else:
+            _vol_clean = math.ceil(volume / 10.0) * 10
+        if volume != _vol_clean:
+            _LOGGER.debug("Volume %s rounded to %s", volume, _vol_clean)
+        return await self._send_message(device, AmazonSequenceType.Volume, _vol_clean)
```

Comment:
```suggestion
        return await self._send_message(device, AmazonSequenceType.Volume, str(_api_volume))
```

and avoid changing the method typing